src/codemagic/apple/resources/enums.py

```diff
@@ -190,13 +189,13 @@ class CertificateType(ResourceEnum):
     MAC_INSTALLER_DISTRIBUTION = 'MAC_INSTALLER_DISTRIBUTION'
 
     @classmethod
-    def from_profile_type(cls, profile_type: ProfileType) -> Optional[CertificateType]:
+    def from_profile_type(cls, profile_type: ProfileType) -> CertificateType:
         if profile_type is profile_type.IOS_APP_ADHOC:
             return CertificateType.IOS_DISTRIBUTION
         elif profile_type is profile_type.IOS_APP_DEVELOPMENT:
             return CertificateType.IOS_DEVELOPMENT
         elif profile_type is profile_type.IOS_APP_STORE:
-            return CertificateType.IOS_DISTRIBUTION
+            return CertificateType.DISTRIBUTION
         elif profile_type is profile_type.MAC_APP_DEVELOPMENT:
             return CertificateType.MAC_APP_DEVELOPMENT
         elif profile_type is profile_type.MAC_APP_STORE:
```

Comment:
macOS is still using `MAC_APP_DISTRIBUTION` by default. It's better to use `DISTRIBUTION` by default instead

Reply:
Resolved in c2686c0. PR description and changelog have also recieved relevant updates.